### ISSUE TITLE
Setup Hero on New Donate Page

### ISSUE THREAD

**Issue by jtfairbank** [distributeaid/next-website-v2#214]
**Design:** [See this Figma file.](https://www.figma.com/design/Yq8LKeIVka701hYL45heud/Colors-for-DA?node-id=181-9119&m=dev)

![Screenshot from 2024-11-12 01 19 54@2x](https://github.com/user-attachments/assets/717aec5e-9654-4907-b778-ddf4985df2a5)

**Code:**

- See `src/app/donate/new-donate` for the space we're working on the new donate page at.
- The current donate page is in `src/app/donate` but this design marks a significant departure from it.
- This component will be reused on multiple pages, so it should get it's own folder in `src/components/` or perhaps be in `src/components/layout`.

**Notes:**

- The buttons link to donate sites (see my comment on the figma file).  Don't worry about the bank transfer popup- we'll tackle that in another issue once this one is done.
- Since we're making a new page with a significantly different design than the old one, there shouldn't need to be too much code reuse.  Let's prioritize using [Radix UI](https://www.radix-ui.com/) components (check both their [Theme](https://www.radix-ui.com/themes/docs/overview/getting-started) and [Primitives](https://www.radix-ui.com/primitives/docs/overview/introduction) docs).   It's ok to sprinkle in some tailwind in order to go above and beyond what we get out of the box with Radix, but don't feel like you need to be pixel-perfect when following the designs.
